src/domain/interfaces/validator.interface.ts

```diff
@@ -1,3 +1,9 @@
+import { Page } from '@/domain/entities/page.entity';
+
 export interface IValidator<T> {
   validate(data: unknown): T;
 }
+
+export interface IPageValidator<T> {
+  validatePage(data: unknown): Page<T>;
+}
```

Comment:
We already have this interface; see `src/domain/interfaces/page-validator.interface.ts`

Reply:
Adjusted in 53082b79be2c075827de2f2c059f984030fbe1b8.